www/apply.shtml

```diff
@@ -18,7 +18,7 @@
           <tr><th><label for="dwname">Your DW username (if you have one):</label></th><td><input type="text" id="dwname" name="dwname" size="25" class="dwuser stretch"></td></tr>
           <tr><th><label for="ljname">Your LJ username (if you have one):</label></th><td><input type="text" id="ljname" name="ljname" size="15" class="ljuser stretch"></td></tr>
           <tr><th><label for="ghname">Your GitHub username:</label></th><td><input type="text" id="ghname" name="ghname" size="25" class="stretch"> (<a href="/why-github.shtml">why do I need a GitHub username?</a>)</td></tr>
-          <tr><th><label for="ircname">Your IRC name (if you use it):</label></th><td><input type="text" id="ircname" name="ircname" size="25" class="stretch"></td></tr>
+          <tr><th><label for="ircname">Your Discord name (if you use it):</label></th><td><input type="text" id="discordname" name="discordname" size="25" class="stretch"></td></tr>
```

Comment:
Looks like `label for="ircname"` should also be updated to `discordname` too, just for consistency.